### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Disk_DeleteDisk_Non-text Contrast: When keyboard focus goes on "Yes" and "No" buttons, the color contrast ratio of border with respect to background is less than 3:1.

### ISSUE THREAD

**Issue by shreyashi227** [microsoft/AzureStorageExplorer#3474]
**GitHubTags:** #C&AI_Aug2020_BMAzure Tools Storage Explorer;#A11yWCAG2.1;#A11YTCS;#DesktopApp;#Azure Tools Storage Explorer;#WCAG1.4.11;#Win32;#A11ySev2;#Benchmark;#ColorContrast;#AI4D;#FTP;

### **Environment Details:**
Application Name: Azure Tool Storage Explorer
Application version: 1.14.2
Windows Version - Windows10

### **Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Delete Disk button and click on it.
7. "Delete Disk" Popup gets opened.
8. Open AI4D and check the color contrast ratio of the border with respect to the background, when the "Yes" and "No" buttons receives the keyboard focus.

### **Actual:**
When the "Yes" and "No" controls receives the keyboard focus, the color contrast ratio of the border with respect to background is less than 3:1.

### **Similar issue is observed for below scenario:**
**1. Disk - Paste Disk**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Navigate to "Disks" of the selected subscription.
5. Navigate to "A11ytest" and  click on it.
6. Navigate to Paste Disk button and click on it.
7. "Paste Disk" Popup gets opened.
8. Open AI4D and check the color contrast ratio of the border with respect to the background, when the "Paste" and "Cancel" buttons receives the keyboard focus.

### **Expected:**
When the "Yes" and "No" control receives the keyboard focus, the color contrast ratio of the border with respect to background should be greater than or equal to 3:1.

### **User Impact:**
Low vision users may not able to see keyboard focus on the control.

### **Attachment:**
https://microsoftapc-my.sharepoint.com/:i:/g/personal/v-shrkh_microsoft_com/EYlxyCaZvGdAqbg6G0Y7hW4BqUVg8jtDNUW1Jua1FDEoVg?e=K1oLNl

**Comment by MRayermannMSFT** [comment]
See screenshot from bug #3463

**Comment by shreyashi227** [comment]
Verified the issue and the issue is not reproducing now so closing this bug.

![Screenshot (276)](https://user-images.githubusercontent.com/60881447/96447983-5ba1e980-1230-11eb-8510-7abe3d9b67de.png)

**Comment by fsteffi** [comment]
**Github Tags:**#A11yMAS;

**Comment by 12-shweta** [comment]
Github Tags:#Closed;
